static/app/utils/replays/replayDataUtils.tsx

```diff
@@ -179,10 +179,13 @@ export function breadcrumbFactory(
     })
     .map(crumb => {
       if (crumb.category === 'replay.mutations') {
+        const crumbData = crumb.data as Record<string, unknown>;
         return {
           ...crumb,
-          type: BreadcrumbType.WARNING,
-          level: BreadcrumbLevelType.WARNING,
+          type: crumbData.limit ? BreadcrumbType.ERROR : BreadcrumbType.WARNING,
```

Comment:
`type` we could explore other options: https://develop.sentry.dev/sdk/event-payloads/breadcrumbs/#breadcrumb-types

Including, creating our own new type `dead-click` which has some cursor logo with a X or something

Reply:
types are really only used here to display the icon in the breadcrumb list iirc, agreed it'd be nice to have a dead click specific one!